db/db_impl/db_impl.h

```diff
@@ -1198,11 +1198,15 @@ class DBImpl : public DB {
   // REQUIRES: db mutex held when calling this function, but the db mutex can
   // be released and re-acquired. Db mutex will be held when the function
   // returns.
-  // After best-efforts recovery, there may be SST files in db/cf paths that are
-  // not referenced in the MANIFEST. We delete these SST files. In the
+  // After recovery, there may be SST files in db/cf paths that are
+  // not referenced in the MANIFEST (e.g. it's best effort recovery,
+  // or the SST files are referenced by unsynced VersionEdits in MANIFEST).
```

Comment:
I feel it's a little confusing to say "unsynced VersionEdits in MANIFEST" in the context of recovery. If the program reaches this point, the prior crash or shutdown has already happened. If a version edit was unsynced before crash, it's very likely that the entry does not exist in the MANIFEST at all.

Reply:
reworded